java-util/src/test/java/io/druid/java/util/common/parsers/TimestampParserTest.java

```diff
@@ -97,8 +97,8 @@ public void testAutoInvalid()
   public void testRuby()
   {
     final Function<Object, DateTime> parser = TimestampParser.createObjectTimestampParser("ruby");
-    Assert.assertEquals(DateTimes.of("2013-01-16T15:41:47+01:00"), parser.apply("1358347307.435447"));
```

Comment:
Will it still parse numbers out of strings?

Reply:
yeah, input in this test case is string.

Reply:
Got it, sorry I missed that there was one in each.